src/utilities/nbackup/nbackup.cpp

```diff
@@ -1430,8 +1431,8 @@ void NBackup::backup_database(int level, Guid& guid, const PathName& fname)
 		Array<UCHAR> unaligned_scns_buffer;
 		Ods::scns_page* scns = NULL, *scns_buf = NULL;
 		{ // scope
-			UCHAR* buf = unaligned_scns_buffer.getBuffer(header->hdr_page_size + SECTOR_ALIGNMENT);
-			scns_buf = reinterpret_cast<Ods::scns_page*>(FB_ALIGN(buf, SECTOR_ALIGNMENT));
+			UCHAR* buf = unaligned_scns_buffer.getBuffer(header->hdr_page_size + sectorSize);
```

Comment:
Wouldn't it be better to use C11 aligned_alloc or C++17 aligned new instead? Page buffers are big enough to be allocated not from pools IMHO.

Reply:
Why it could be better ? 
There are tens usages of `FB_ALIGN` in the code, do you offer to replace its all in this PR ? 
Also, do you offer to replace RAII class `Array` by `alloc` \ `new` and (a lot of)  `free` \ `delete`'s ?

Reply:
No. Only in places where aligned memory is needed. And yes, replace it with RAII std::unique_ptr.

Reply:
I see no reason to do it in this PR.